web_src/js/modules/tippy.js

```diff
@@ -30,7 +27,8 @@ export function createTippy(target, opts = {}) {
       visibleInstances.add(instance);
     },
     arrow: `<svg width="16" height="7"><path d="m0 7 8-7 8 7Z" class="tippy-svg-arrow-outer"/><path d="m0 8 8-7 8 7Z" class="tippy-svg-arrow-inner"/></svg>`,
-    ...(opts?.role && {theme: opts.role}),
+    role: 'menu', // HTML role attribute, only tooltips should use "tooltip"
+    theme: 'menu', // CSS theme, we support either "tooltip" or "menu"
```

Comment:
But now what you do is more or less like what I did before .... 

`createTippy({role: 'menu'})` or `createTippy({role: 'tooltip'})` won't have default "role" theme now 🤣

Reply:
Hmm I guess I can add that back again, not that it matters really :)

Reply:
And what if there are other roles in the future (say, combobox list ?), defaulting "theme" to "menu" seems strange.

Reply:
Done in https://github.com/go-gitea/gitea/pull/25134/commits/ab6965b7b1f04a8a27f2b1e65b1c2681920744d3.